### PR TITLE
LPD-34681 | LPD-34682 Add new fields to the refund detail page

### PR DESCRIPTION
**Pull request by ghost**
https://liferay.atlassian.net/browse/LPD-34681
https://liferay.atlassian.net/browse/LPD-34682

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -14811,6 +14812,7 @@ refund-status=Refund Status
 refund-subtotal=Refund Subtotal
 refund-summary=Refund Summary
 refunded=Refunded
+refunded-by=Refunded by
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14815]: We should capitalize "by", `Refunded By`

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Pull request closed without merging**